app/routes/events/components/EventAdministrate/index.tsx

```diff
@@ -35,7 +35,12 @@ const EventAdministrateIndex = (props: Props) => {
         }}
       >
         <NavigationLink to={`${base}/attendees`}>Påmeldinger</NavigationLink>
-        <NavigationLink to={`${base}/allergies`}>Allergier</NavigationLink>
+        {(props.currentUser.id == props.event.createdBy ||
+          props.currentUser == props.event.createdBy) &&
+          props.currentUser.id &&
+          props.event.createdBy && (
```

Comment:
Wait is `props.event.createdBy` either a user id or a user object? That's kinda dumb

Reply:
Yes, when running local backend its just an integer, but in prod its a user object..